spec/services/solidus_bolt/transaction/capture_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+# rubocop:disable RSpec/MultipleMemoizedHelpers
+# rubocop:disable RSpec/SubjectStub
```

Comment:
It looks to me we can move this cop inline [here](https://github.com/nebulab/solidus_bolt/pull/21/commits/441b1ea1ded5a6ac8d62b1439b0cea79084d3678#diff-e2145e5b0d096b767c2df154f5bf676ad894d234857b4dba9a79d89cbce38b14R28)

Reply:
moved inline